docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
       dockerfile: openwisp_dashboard/Dockerfile
       args:
         DASHBOARD_APP_PORT: 8000
+        OPENWISP_MONITORING_SOURCE: https://github.com/openwisp/openwisp-monitoring/tarball/master
```

Comment:
@nemesisdesign the docker image is installing the version from PyPI even when I have specified this here. 
I even tried creating `.build.env` locally. But, it had no effect.